.github/contributing.md

```diff
@@ -12,8 +12,8 @@ Make sure your pull request follows the guidelines/checklists in this document.
 - Double-check spelling and grammar.
 - No trailing whitespace is added to the end of any file.
 - One suggestion per pull request.
-- Entries that repeatedly violate guidelines will be removed!
-- You have to [sign your commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).
+- I understand that my entry will be removed if it violates the guidelines.
```

Comment:
Imo it doesn't make much sense to word it like this in the guidelines. You're not affirming anything in this context. This is purely informational.

It's also weird that the guidelines change perspective from one line to the next (going from _you have_ to _I understand_)

Reply:
True, got any better suggestion? Maybe it could all be changed to a literal checklist, or just point to the PR template to completely avoid duplication